pom.xml

```diff
@@ -227,8 +227,8 @@
         <artifactId>maven-compiler-plugin</artifactId>
         <version>3.1</version>
         <configuration>
-          <source>1.7</source>
-          <target>1.7</target>
+          <source>1.8</source>
+          <target>1.8</target>
```

Comment:
@jongyoul  Yes I will revert that value to 1.7 with the next commit. But for this ipfs code to execute the mvn clean install must be with jsk 1.8.